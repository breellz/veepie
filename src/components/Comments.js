import React from 'react';

const Comments = () => (
  <div>
    Comments page content
  </div>
);

export default Comments;
