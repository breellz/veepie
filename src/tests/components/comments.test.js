import React from 'react';
import { shallow } from 'enzyme'
import Comments from '../../components/Comments'

test('should render comments correctly', ()=>{
    const renderer = shallow(<Comments />)
    expect(renderer).toMatchSnapshot()
})