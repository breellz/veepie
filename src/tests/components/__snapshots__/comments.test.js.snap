// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render comments correctly 1`] = `
<div>
  <form>
    <textarea
      placeholder="your comment goes here"
    />
    <button>
      Post Comment
    </button>
  </form>
</div>
`;
